test/fixtures/cookbooks/test/recipes/filter_lines.rb

```diff
@@ -0,0 +1,125 @@
+directory '/tmp'
+
+template '/tmp/dangerfile' do
+end
+
```

Comment:
Some examples of what's possible.  I haven't written the inspec tests yet.  Coming up next.